service/health/searcher/service_test.go

```diff
@@ -0,0 +1,76 @@
+package searcher
+
+import (
+	"strconv"
+	"testing"
+
+	apiextensionsfake "github.com/giantswarm/apiextensions/pkg/clientset/versioned/fake"
+	"github.com/giantswarm/micrologger/microloggertest"
+)
+
+func Test_Health_New(t *testing.T) {
```

Comment:
I'd skip this test. IMO testing `IsInvalidConfig` introduces more clutter than value.

Reply:
I am personally fine with having them here, as we won't have too many services in this microservice.
So if anything I think we can keep it here and still get rid of it later if it gets annoying.

Reply:
It currently provides a small amount of value and can be extended in the future if we come up with more tests so I think I might keep it and, as Marcel said, we can get rid of it in the future if it seems useless.